app/src/androidTest/java/com/sdp/swiftwallet/UiTest/MainActivityTest.java

```diff
@@ -48,11 +48,32 @@ public void bottom_bar_with_icons_is_displayed() {
     }
 
     @Test
-    public void click_display_fragments() {
+    public void press_home_display_fragments() {
         onView(withId(R.id.bar_home)).perform(click());
+        onView(withId(R.id.home_fragment)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void press_stats_display_fragments() {
         onView(withId(R.id.bar_stats)).perform(click());
+        onView(withId(R.id.stats_fragment)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void press_payment_display_fragments() {
```

Comment:
A detail but isn't the mixed case naming convention cleaner ?
```suggestion
    public void pressPaymentDisplaysFragments() {
```